code/src/components/TodoList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { TodoInput } from './TodoInput.js'
+import { TodoItem } from './TodoItem.js'
+import { TodoSummary } from './TodoSummary.js'
+import { ClearAllButton } from 'lib/ClearAllButton.js'
+import styled from 'styled-components/macro'
+
+export const TodoList = () => {
+  const list = useSelector((store) => store.todos.list)
```

Comment:
Same here, since you're only using the items-array, get only that:
`const items = useSelector((store) => store.todos.list.items)`